### PR TITLE
Bugfix: Correctly detect message name in FinderInvokeStrategy and Han…

### PR DESCRIPTION
**Pull request by prolic**
…dleCommandStrategy

based on: https://github.com/prooph/service-bus/pull/108

### CODE REVIEW

File: tests/Plugin/InvokeStrategy/FinderInvokeStrategyTest.php [hunk 1]
```diff
@@ -63,4 +64,20 @@ public function it_invokes_a_finder_which_has_method_named_like_the_query()
         $this->assertSame($this->actionEvent->getParam(QueryBus::EVENT_PARAM_DEFERRED), $finder->getLastDeferred());
         $this->assertTrue($this->actionEvent->getParam(QueryBus::EVENT_PARAM_MESSAGE_HANDLED));
     }
+
+    /**
+     * @test
+     */
+    public function it_determines_the_event_name_from_message_name_call_if_event_has_one()
+    {
+        $onEventStrategy = new FinderInvokeStrategy();
```
Comment (codeliner) [inline tests/Plugin/InvokeStrategy/FinderInvokeStrategyTest.php]: s/$onEventStrategy/$finderInvokeStrategy

File: tests/Plugin/InvokeStrategy/FinderInvokeStrategyTest.php [hunk 2]
```diff
@@ -63,4 +64,20 @@ public function it_invokes_a_finder_which_has_method_named_like_the_query()
         $this->assertSame($this->actionEvent->getParam(QueryBus::EVENT_PARAM_DEFERRED), $finder->getLastDeferred());
         $this->assertTrue($this->actionEvent->getParam(QueryBus::EVENT_PARAM_MESSAGE_HANDLED));
     }
+
+    /**
+     * @test
+     */
+    public function it_determines_the_event_name_from_message_name_call_if_event_has_one()
```
Comment (codeliner) [inline tests/Plugin/InvokeStrategy/FinderInvokeStrategyTest.php]: s/the_event_name/the_query_name

File: tests/Plugin/InvokeStrategy/HandleCommandStrategyTest.php [hunk 3]
```diff
@@ -60,4 +61,20 @@ public function it_invokes_the_handle_command_method_of_the_handler_without_comm
 
         $this->assertSame($doSomething, $handleCommandHandler->getLastMessage());
     }
+
+    /**
+     * @test
+     */
+    public function it_determines_the_event_name_from_message_name_call_if_event_has_one()
```
Comment (codeliner) [inline tests/Plugin/InvokeStrategy/HandleCommandStrategyTest.php]: s/the_event_name/the_command_name

File: tests/Plugin/InvokeStrategy/HandleCommandStrategyTest.php [hunk 4]
```diff
@@ -60,4 +61,20 @@ public function it_invokes_the_handle_command_method_of_the_handler_without_comm
...
+    {
+        $onEventStrategy = new HandleCommandStrategy();
```
Comment (codeliner) [inline tests/Plugin/InvokeStrategy/HandleCommandStrategyTest.php]: s/$onEventStrategy/$onCommandStrategy

**Comment by prolic** [comment]
@codeliner merge?

**Comment by codeliner** [comment]
@prolic sorry, of course. bugfix release before we continue?

**Comment by prolic** [comment]
Yes bugfix release sounds good
Am 09.02.2016 07:26 schrieb "Alexander Miertsch" <notifications@github.com>:

**Pull request merged**